lib/vm.js

```diff
@@ -85,14 +85,38 @@ function validateString(prop, propName) {
                                'string', prop);
 }
 
+function validateBool(prop, propName) {
+  if (prop !== undefined && typeof prop !== 'boolean')
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', propName,
+                               'boolean', prop);
+}
+
+function validateObject(prop, propName) {
+  if (prop !== undefined && prop !== null && typeof prop !== 'object')
```

Comment:
Sorry for keeping going back and forth on this, but we don’t want to allow `null` here right?

Reply:
yea that seems right. I think I accidentally inverted the standard object type check in my head which is why the null is there, I'll fix it